docs/Advanced/Refs.mdx

```diff
@@ -69,14 +69,31 @@ export interface CanvasRef {
   containerHeight?: number;
 
   /**
-   * Center the canvas to the viewport.
+   * Positions the canvas to the viewport.
    */
-  centerCanvas?: () => void;
+  positionCanvas?: (position: CanvasPosition, animated?: boolean) => void;
```

Comment:
How is `positionCanvas` different from `setScrollXY`? Wouldn't they technically be the same thing?

Reply:
This is another one I pasted over into these outdated docs

`positionCanvas` does a `setXY` and a `setScrollXY` https://github.com/reaviz/reaflow/blob/28047d52acce5154ffce58dd394b0798df9732f3/src/layout/useLayout.ts#L189-L195